packages/web/components/complex/all-pools-table-set.tsx

```diff
@@ -51,70 +42,74 @@ type PoolWithMetrics = {
   feesPercentage: string;
 };
 
+export type Pool = [
+  {
+    poolId: string;
+    poolAssets: { coinImageUrl: string | undefined; coinDenom: string }[];
+    stableswapPool: boolean;
+  },
+  {
+    value: PricePretty;
+  },
+  {
+    value: PricePretty;
+    isLoading?: boolean;
+  },
+  {
+    value: PricePretty;
+    isLoading?: boolean;
+  },
+  {
+    value: PricePretty | RatePretty | undefined;
+    isLoading?: boolean;
+  },
+  {
+    poolId: string;
+    cellGroupEventEmitter: EventEmitter<string | symbol, any>;
+    onAddLiquidity?: () => void;
+    onRemoveLiquidity?: () => void;
+    onLockTokens?: () => void;
+  }
+];
+
+type Filter = "superfluid" | "stable" | "concentrated" | "weighted";
```

Comment:
IMO if this type isn't needed elsewhere, it can just be removed. Generally, we should only add types if they're beneficial elsewhere, as a way to minimize code &/ complexity.